### PR TITLE
UCI Nginx Config - Remove rewrite rules

### PR DESCRIPTION
**Pull request by ChakshuGautam**

### CODE REVIEW

**Comment by ChakshuGautam** [comment]
@keshavprasadms please review.

File: kubernetes/helm_charts/core/nginx-private-ingress/templates/configmap.yaml [hunk 1]
```diff
@@ -174,17 +174,17 @@ data:
         }
         location /gql-service/ {
           set $target http://gql-service.{{ .Values.namespace }}.svc.cluster.local:8080;
-          rewrite ^/gql-service/(.*) /$1 break;
+          rewrite ^/(.*) /$1 break;
```
Comment (rjshrjndrn) [inline kubernetes/helm_charts/core/nginx-private-ingress/templates/configmap.yaml:177]: I don't think we need this rewrite block, as we're replacing the same string. like, replace apple with apple. So it'll be good to drop all these rewrites.

**Review by rjshrjndrn — CHANGES_REQUESTED** [review]

File: kubernetes/helm_charts/core/nginx-private-ingress/templates/configmap.yaml (hunk 1 above)
Comment (ChakshuGautam) [reply kubernetes/helm_charts/core/nginx-private-ingress/templates/configmap.yaml:177]: Great thanks. Will just update this.

**Comment by ChakshuGautam** [comment]
@rjshrjndrn Please check.

**Review by keshavprasadms — APPROVED** [review]

**Comment by ChakshuGautam** [comment]
@keshavprasadms @rjshrjndrn Currently the `http://11.2.6.6/gql-service/console/` is getting translated to `/gql-service/console/` but I need it to be `/console/`. What changed do I need to make this routing work?

**Pull request merged**